query/src/test.rs

```diff
@@ -500,14 +459,56 @@ impl TestChunk {
         }
     }
 
-    pub fn with_table(mut self, name: impl Into<String>) -> Self {
-        self.table_names.push(name.into());
+    /// specify that any call should result in an error with the message
+    /// specified
+    pub fn with_error(mut self, error_message: impl Into<String>) -> Self {
+        self.saved_error = Some(error_message.into());
         self
     }
 
-    /// Get a copy of any predicate passed to table_names
-    pub fn table_names_predicate(&self) -> Option<Predicate> {
-        self.table_names_predicate
+    /// Checks the saved error, and returns it if any, otherwise returns OK
+    fn check_error(&self) -> Result<()> {
+        if let Some(message) = self.saved_error.as_ref() {
+            General { message }.fail()
+        } else {
+            Ok(())
+        }
+    }
+
+    /// Register a table with the test chunk and a "dummy" column
+    pub fn with_table(self, table_name: impl Into<String>) -> Self {
+        self.with_tag_column(table_name, "dummy_col")
+    }
+
+    /// Register an int tag column with the test chunk
```

Comment:
```suggestion
    /// Register an int tag column with the test chunk
```

What's an "int tag column"? I thought tags were always `String`? Or does "int" here mean "internal"...? Or something else?

Reply:
You are correct (again!). This should not refer to `int` at all. Just `tag`. I will fix

```suggestion
    /// Register a tag column with the test chunk
```

Reply:
fixed in bdc98b0a